Makefile

```diff
@@ -133,11 +140,10 @@ doc: crdoc ## Generate
 	$(CRDOC) --resources config/crd/bases/flows.netobserv.io_flowcollectors.yaml --output docs/FlowCollector.md
 
 generate-go-conversions: $(CONVERSION_GEN) ## Run all generate-go-conversions
-		$(MAKE) clean-generated-conversions SRC_DIRS="./api/v1alpha1"
 		$(CONVERSION_GEN) \
 		--input-dirs=./api/v1alpha1 \
 		--build-tag=ignore_autogenerated_core \
-		--output-file-base=zz_generated.conversion \
+		--output-file-base=zz_generated.conversion $(CONVERSION_GEN_OUTPUT_BASE)\
```

Comment:
We had to push this to fix `conversion_gen` missing file generating lint error.

For now, [I have disabled the conversion](https://github.com/netobserv/network-observability-operator/pull/252/files#diff-23e9a27bd609a9414ced41fdaf8063ade03a51b3154d85ad4edfdae228dd5531R45) and added the new fields in both v1alpha1 and v1beta1 apis

Reply:
I am looking at this and also created an issue for the conversion-gen tool 
https://github.com/kubernetes/code-generator/issues/151

Reply:
@jpinsonneau pls include this change to fix the conversion issue
https://github.com/netobserv/network-observability-operator/pull/268/commits/579cd3d525736634a72c703e023332417bb62d48#diff-23e9a27bd609a9414ced41fdaf8063ade03a51b3154d85ad4edfdae228dd5531

Reply:
https://github.com/netobserv/network-observability-operator/pull/252/commits/4a689b594c52a36ddd292e0303d3fd5e02ad6130 here we go, thanks @msherif1234 !